app/_data/tables/version_errors_konnect.yml

```
@@ -499,3 +988,401 @@ messages:
+      as intended.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.1` or above.
+    DocumentationURL: |
+      [Zipkin plugin](/hub/kong-inc/zipkin/)
+  - ID: P137
+    Severity: error
+    Description: |
+      For the `rate-limiting` plugin, one or more of the following `config` fields are
+      set: `error_code`, `error_message` but Kong Gateway versions < 3.1 do not
+      support these fields. Plugin features that rely on these fields are not working
+      as intended.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.1` or above.
+    DocumentationURL: |
+      [Rate Limiting plugin](/hub/kong-inc/rate-limiting/)
+  - ID: P138
+    Severity: error
+    Description: |
+      For the `acme` plugin, one or more of the following `config` fields are set:
+      `storage_config.redis.ssl`, `storage_config.redis.ssl_verify`,
+      `storage_config.redis.ssl_server_name` but Kong Gateway versions < 3.1 do not
+      support these fields. Plugin features that rely on these fields are not working
+      as intended.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.1` or above.
+    DocumentationURL: |
+      [ACME plugin](/hub/kong-inc/acme/)
+  - ID: P139
+    Severity: error
+    Description: |
+      For the `response-ratelimiting` plugin, one or more of the following `config`
+      fields are set: `redis_ssl`, `redis_ssl_verify`, `redis_server_name` but Kong
+      Gateway versions < 3.1 do not support these fields. Plugin features that rely on
+      these fields are not working as intended.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.1` or above.
+    DocumentationURL: |
+      [Response Ratelimiting plugin](/hub/kong-inc/response-ratelimiting/)
+  - ID: P140
+    Severity: error
+    Description: |
+      For the `aws-lambda` plugin, one or more of the following `config` fields are
+      set: `aws_assume_role_arn`, `aws_role_session_name` but Kong Gateway versions <
+      3.0 do not support these fields. Plugin features that rely on these fields are
+      not working as intended.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.0` or above.
+    DocumentationURL: |
+      [AWS Lambda plugin](/hub/kong-inc/aws-lambda/)
+  - ID: P141
+    Severity: error
+    Description: |
+      The `key` entity is being used, but Kong Gateway versions < 3.1 do not support
+      this entity.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.1` or above.
+    DocumentationURL: |
+      [Keys entity](/gateway/latest/admin-api/#keys-entity)
+  - ID: P142
+    Severity: error
+    Description: |
+      The `key-set` entity is being used, but Kong Gateway versions < 3.1 do not
+      support this entity.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.1` or above.
+    DocumentationURL: |
+      [Key Sets entity](/gateway/latest/admin-api/#key-sets-entity)
```

```suggestion
      [Key Sets entity](/gateway/latest/admin-api/#keys-object)
```
We don't seem to have a separate section for key sets, they're just part of keys.